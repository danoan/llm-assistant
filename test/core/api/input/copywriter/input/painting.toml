input_text="We could say a painting made by a two years old monkey"
