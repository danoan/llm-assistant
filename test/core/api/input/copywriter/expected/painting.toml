content="The artwork appears to lack depth and sophistication, resembling the work of a young and inexperienced artist."
